code/modules/projectiles/projectile/bullets/rifle.dm

```diff
@@ -228,3 +228,51 @@
 	desc = "doink!"
 	damage_type = BRUTE
 	icon_state = "paperball"
+
+//Big Ballista Mounted gun Spear
+/obj/projectile/bullet/Large_Ballista_Spear
```

Comment:
```suggestion
/obj/projectile/bullet/ballista_spear
```
Don't capitalise type names
Also, you probably don't need to specify it's large when there's no smaller kind of ballista spear
to be honest just `bullet/ballista` would probably even be fine.